docs/guides/high-availability/examples/generated/keycloak.yaml

```diff
@@ -470,29 +476,9 @@ spec:
                     name: keycloak-preconfigured-admin
                     key: password
                     optional: false
-                # tag::keycloak[]
               - name: JAVA_OPTS_APPEND # <5>
-                value: >
-                  -Xms512m -Xmx1524m
```

Comment:
@ahus1 are we not going to propagate the heap settings in the new template ?

Reply:
The heap settings are derived from the JVM automatically using percentages of the memory limit. See

https://github.com/keycloak/keycloak/blob/59007844d9978daf433f684889dd90036ff75ca3/quarkus/dist/src/main/content/bin/kc.sh#L101-L112

for the code. Therefore, there is no need to pass on heap settings to the JVM. And this is also the reason for the rather complicated computation of limit and request.